src/main/bin/ccm-initialise

```diff
@@ -128,9 +127,9 @@ MakeCacheRoot($this_app, $cache_root, $dopts);
 
 # make global lock file (mask and GID take care of the permissions)
 $this_app->verbose("Creating lockfile");
-open TMP, ">$cache_root/global.lock";
-print TMP "no\n";
-close TMP;
+open my $TMP, ">$cache_root/global.lock";
```

Comment:
I think severity 1 is probably too insane, at that level you get into some subjective/really unlikely stuff. Severity 5 picks up on the stuff that causes bugs that are hard to track down or have catastrophic impact in rare cases, and it's not too difficult to get there. (But obviously great we can activate policies one by one as we decide we agree with them or not.)